prep-javascript-variables/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    var fullName = 'Nicholas Cardoza';
+    console.log(fullName);
+    var numberOfPets = '2';
+    console.log(numberOfPets);
```

Comment:
We'll want to leave the quotes off of our number values so that the javascript engine doesn't interpret them as strings